v-next/hardhat-ignition-viem/eslint.config.cjs

```diff
@@ -0,0 +1,16 @@
+const { createConfig } = require("../../config-v-next/eslint.config.cjs");
+
+const configs = createConfig(__filename);
+
+/**
+ * @type {import("eslint").Linter.Config}
+ */
+const overrideConfig = {
+  ignores: ['/test/fixture-projects'],
```

Comment:
remember to remove this line once you moved the ignore pattern to the base config